Organiser.Core/Organiser.Core/Context/DataBaseContext.cs

```diff
@@ -87,6 +87,18 @@ public void CreateOrUpdate(Savings saving)
         public IQueryable<Savings> AllSavings => dataContext.Savings;
         #endregion
 
+        #region Bugs
+        public IQueryable<Bugs> Bugs => dataContext.Bugs.Where(x => x.BUID == user.UID);
+        public IQueryable<Bugs> AllBugs => dataContext.Bugs;
```

Comment:
there is a repository pattern for defining queries. I'd suggest to abstract away these queries to repository class. it helps with separation of concerns and single responsibility principles